Project.toml

```diff
@@ -12,6 +12,7 @@ DataFramesMeta = "1313f7d8-7da2-5740-9ea0-a2ca25f37964"
 EnumX = "4e289a0a-7415-4d19-859d-a7e5c4648b56"
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MolecularGraph = "6c89ec66-9cd8-5372-9f91-fabc50dd27fd"
```

Comment:
The corresponding compat entry is missing.

Reply:
fixed.